src/component/handlers/composition/DraftEditorCompositionHandler.js

```diff
@@ -45,7 +45,12 @@ let textInputData = '';
 
 var DraftEditorCompositionHandler = {
   onBeforeInput: function(editor: DraftEditor, e: SyntheticInputEvent): void {
-    textInputData = (textInputData || '') + e.data;
+    // If insertText event type fires here, that means we're at the end of a composition
```

Comment:
Is this still relevant?

Reply:
I'm gonna keep it for now but will remove if i can in a later pr, this one is a little more nuanced iirc